device_policy.py

```diff
@@ -9,7 +9,7 @@
 
 import signer
 
-#tools for working with the device policy blobs
+# Tools for working with the device policy blobs.
```

Comment:
Reformatting all of the comments is pointless.

Reply:
I see no reason not to